src/server/api/medias/musicres.js

```diff
@@ -10,7 +10,7 @@ router.get('/', (req, res) => {
     return res.status(403).send(new Error('Unauthorized role'));
   }
   const { label } = res.locals.context.context;
```

Comment:
I would prefer leave it as it is now because most course doesn't have music reserves and hardcoding a sample allows testing.

Reply:
@maxpluez I disagree. You should build this course in your Moodle then: https://ccle.ucla.edu/mod/qanda/view.php?id=897711&mode=entry&hook=3041

Or change the shortname of the course you are testing with to match this rather than edit the code.